### PR TITLE
refactor the return message

### PR DESCRIPTION
**Pull request by DaniMB95**

### CODE REVIEW

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java [hunk 1]
```diff
@@ -81,8 +82,9 @@ public static void faceDetection (String file, String type) throws FaceDetection
             //Save the image already processed with the face detection
             Imgcodecs.imwrite(currentDir + "\\src\\main\\resources\\images\\" + finalFile, src);            //logger: info that everything it right
             log.info("It's only one person in the image");
+            status = true; //status of the face detection
             //result if the image it's a profile
-            result = "profile detected";//"";
+            result = "it's a person: " + status;//"";
```
Comment (saraialvarezrosas) [inline searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java]: Please remove this //"";      or add an explanation of what it is for.

**Review by rodri-98 — APPROVED** [review]
Good job!

**Review by saraialvarezrosas — APPROVED** [review]

File: searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java (hunk 1 above)
Comment (DaniMB95) [reply searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java]: already erase the "//"

**Review by FernandezRobert — APPROVED** [review]

**Review by FernandezRobert — CHANGES_REQUESTED** [review]
Fix conflicts please

**Pull request closed without merging**